.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,214 @@
+name: Benchmarks 
+on:
+  push:
+    branches:
+      - main
+      - custom-dashboarding
+  workflow_dispatch:
+
+concurrency:
+  # Fairly strict concurrency rule to avoid stepping on benchmark db.
+  # Could eventually replace with a real db in coiled, RDS, or litestream
+  group: benchmarks
+  cancel-in-progress: true
+
+defaults:
+  # Required shell entrypoint to have properly activated conda environments
+  run:
+    shell: bash -l {0}
+
+jobs:
+  software:
+    name: Setup
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.9"]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up environment
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniforge-variant: Mambaforge
+          use-mamba: true
+          python-version: ${{ matrix.python-version }}
+          environment-file: ci/environment.yml
+
+      - name: Build Coiled Software Environment
```

Comment:
Yes, I'd be very happy to avoid this step! I don't have the strongest grasp on the nightly images and what their lifecycle is, so any suggestions for how to structure that are welcome